### PR TITLE
Set actorsv2 upgrade epoch

### PR DESCRIPTION
**Pull request by arajasek**
Ready to merge

### CODE REVIEW

**Review by Stebalien — APPROVED** [review]

**Review by magik6k — APPROVED** [review]

**Review by magik6k — APPROVED** [review]

**Review by Kubuxu — APPROVED** [review]
SGWM

File: build/params_2k.go
Comment (momack2) [inline build/params_2k.go:18]: I notice liftoff changed from -3 to -4 -- does that need to change anywhere else?

Comment (Stebalien) [reply build/params_2k.go:18]: These can be any negative number (this file is only built for testing).

Comment (arajasek) [reply build/params_2k.go:18]: The other place would be build/params_testground.go, which is already done

**Review by whyrusleeping — APPROVED** [review]
LGTM

**Review by magik6k — APPROVED** [review]

File: chain/stmgr/forks.go
Comment (jbenet) [inline chain/stmgr/forks.go:561]: I think it would be great practice to assert the changes happened right after performing them. it's the type of defensive programming that belongs in code where correctness is paramount. In this case that would be asserting `StartEpoch`, `UnlockDuration`, and `InitialBalance` are indeed 0 for these three msigs.

File: chain/stmgr/forks.go
Comment (anorth) [inline chain/stmgr/forks.go:839]: Nit: I think this is ok in practise, but note that blindly setting these can break actor state invariants. The state that these msig actors are in now, with nonzero StartEpoch but zero UnlockDuration is in fact impossible to reach by normal means. It's only possible because the prior upgrade manipulated state directly, into a configuration that the actor code is actually not prepared to handle. The code, for better or worse, expects nonzero `InitialBalance` and `StartEpoch` iff `UnlockDuration!=0`. See the actor constructor and `LockBalance` method.

I *think* that nothing bad will come of it here (except the undesirable lockup that this change reverts), but it's not a good practise. The "impossible" state configurations can have no test coverage in the actors project.

In this PR, the callers all seem to set `(0, 0, 0)`, which is a supported configuration.

For the future, I recommend:
- careful study of the code of actors which state you intend to manipulate directly
- extraction and maintenance of the same invariants enforced by the actor code
- requiring review from an actors maintainer
- [actors team] pushing invariants lower down the call tree so they're easier to include by default

**Pull request merged**
